### PR TITLE
WFP-1694

### PR DESCRIPTION
**Pull request by carlov20**

### CODE REVIEW

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt [hunk 1]
```diff
@@ -60,9 +48,7 @@ class GetCaseByCrnTests : IntegrationTestBase() {
       .jsonPath("$.requirements[0].subCategory")
       .isEqualTo("Regular")
       .jsonPath("$.requirements[0].length")
-      .isEqualTo(100)
-      .jsonPath("$.requirements[0].lengthUnit")
```
Comment (carlov20) [inline src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt:64]: the only breaking change, have raised https://github.com/ministryofjustice/manage-a-workforce-ui/pull/629 to move to the new field

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/dto/DeliusCaseView.kt [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+  val streetName: String?,
+  val town: String?,
+  val county: String?,
+  val postcode: String?,
+  val noFixedAbode: Boolean,
+  val typeVerified: Boolean,
+  val typeDescription: String,
+  val startDate: LocalDate
+) {
+  val from: LocalDate
```
Comment (carlov20) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/dto/DeliusCaseView.kt:33]: to be backwards compatible

File: src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/dto/DeliusCaseView.kt [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+    get() = startDate
+  val type: MainAddressType
```
Comment (carlov20) [inline src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/client/dto/DeliusCaseView.kt:35]: to be backwards compatible

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt (hunk 1 above)
Comment (sp-watson) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt:64]: So these have to be released together?

**Review by sp-watson — APPROVED** [review]

File: src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt (hunk 1 above)
Comment (carlov20) [reply src/test/kotlin/uk/gov/justice/digital/hmpps/hmppsallocations/integration/unallocatedcases/GetCaseByCrnTests.kt:64]: I want to see what just releasing this does in dev, not sure how nunjucks will handle nulls

**Pull request merged**